src/API/HealthController.ts

```diff
@@ -0,0 +1,28 @@
+import { inject, injectable } from 'inversify'
+import { Db } from 'mongodb'
+
+import { IPFS } from './IPFS'
+
+@injectable()
+export class HealthController {
+  private readonly db: Db
+  private readonly ipfs: IPFS
+
+  constructor(@inject('DB') db: Db, @inject('IPFS') ipfs: IPFS) {
+    this.db = db
+    this.ipfs = ipfs
+  }
+
+  async getHealth(): Promise<any> {
+    const mongoConnection = await this.db.stats()
```

Comment:
Each function should do 1 thing. You can move the mongo and ipfs logic into their own private methods. getHealth should only be composing together the results of its own private methods, it shouldn't know about external stuff.

```
async getHealth(): Promise<any> {
  const mongoStatus = this.getMongoStatus()
  const ipfsStatus = this.getIPFSStatus()
  // ...stuff
}
```